Dockerfile

```diff
@@ -24,11 +24,23 @@ COPY /app .
 
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o /go/bin/mlpab
 
-FROM alpine:3.16.1
+FROM build-env as development
+
+WORKDIR /go/bin
+COPY --from=build-env /app .
+COPY --from=asset-env /app/web/static web/static
+COPY web/template web/template
+
+# Live reload for Go
+RUN go install github.com/cosmtrek/air@latest
```

Comment:
if you moved this above the `COPY`s it would be able to cache the layer (I think)

Reply:
thanks, moved.